sdcm/utils/argus.py

```diff
@@ -26,7 +26,7 @@ def is_uuid(uuid) -> bool:
     try:
         UUID(uuid)
         return True
-    except (ValueError, AttributeError):
+    except (ValueError, AttributeError, TypeError):
```

Comment:
The same here. Looks like the commit `4` could be squashed into the `3`rd one.

Reply:
Done